test/serializers/attribute_test.rb

```diff
@@ -71,6 +71,21 @@ def id
 
         assert_equal('custom', hash[:blog][:id])
       end
+
+      PostWithVirtualAttribute = Class.new(::Model)
+      class PostWithVirtualAttributeSerializer < ActiveModel::Serializer
+        attribute :name do
+          "#{object.first_name} #{object.last_name}"
+        end
+      end
```

Comment:
I should add a case here using the `virtual_attribute` option.  e.g. the poro in our fixtures file
```ruby
class VirtualValueSerializer < ActiveModel::Serializer
  attributes :id

  has_many :reviews, virtual_value: [{ id: 1 }, { id: 2 }]
end
```